### PR TITLE
Added timestamps to restore process

### PR DESCRIPTION
**Pull request by jcpunk**
This should make it easier to see how long a given restore took.

### CODE REVIEW

**Comment by ehelms** [comment]
[test]

**Comment by ehelms** [comment]
@cfouant can you please review?

**Comment by jlsherrill** [comment]
[test]

File: katello/katello-restore [hunk 1]
```diff
@@ -91,7 +92,11 @@ def confirm
   print "Are you sure(Y/N)? "
   response = gets.chomp
   if /[Y]/i.match(response)
+    @time = Time.now
+    puts "Starting restore from #{DIR}: #{@time}"
```
Comment (cfouant) [inline katello/katello-restore]: nitpick: consolidate code with `puts "Starting restore from #{DIR}: #{Time.now}"` and remove previous line

File: katello/katello-restore [hunk 2]
```diff
@@ -91,7 +92,11 @@ def confirm
...
     restore
+    @time = Time.now
+    puts "Done with restore: #{@time}"
```
Comment (cfouant) [inline katello/katello-restore]: nitpick: consolidate code with `puts "Done with restore: #{Time.now}"` and remove previous line

**Comment by jcpunk** [comment]
Changes applied

**Comment by cfouant** [comment]
ACK pending green tests! Thanks @jcpunk!

**Comment by ehelms** [comment]
[test]

**Comment by cfouant** [comment]
@ehelms - can you merge plz and thank you?

**Pull request merged**